app/src/main/java/net/gsantner/markor/format/ActionButtonBase.java

```diff
@@ -58,32 +58,134 @@
 import java.util.LinkedHashSet;
 import java.util.List;
 import java.util.Map;
+import java.util.Objects;
 import java.util.Set;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 @SuppressWarnings({"WeakerAccess", "UnusedReturnValue"})
 public abstract class ActionButtonBase {
-    private Activity m_activity;
-    private MarkorContextUtils m_cu;
+    public static final String ACTION_ORDER_PREF_NAME = "action_order";
```

Comment:
it looks like the auto format, I was asked to use, changed a lot of formattings. And it reordered the import section

Reply:
>  And it reordered the import section

Please use Android Studio and it's auto-reformatting, or disable auto-reformatting if you can't. Otherwise one other developer runs auto-format as well and there are again many lines changes for no reason.

Rather running one time a auto-reformat on your whole PR before merging, than trying to review and compare only reformat changes.

Reply:
I reverted the changes from Auto-Format
How this should be handled in the future?

Reply:
>  And it reordered the import section

Please use Android Studio and it's auto-reformatting, or disable auto-reformatting if you can't. Otherwise one other developer runs auto-format as well and there are again many lines changes for no reason.

Rather running one time a auto-reformat on your whole PR before merging, than trying to review and compare only reformat changes.

Reply:
>  How this should be handled in the future?

* If you prefer to work with visual studio, finding a way that formats in the same way like android studio (there is also some commandline utitlity, but I never tried that) 
* Or, disable auto-reformatting generally, and we once do a reformat on all files prior merge.

Reply:
It just makes it hard to review when there are changes where you actually didn't change anything at all - but still the reviewer would need to check it as well, because reviewer wouldn't know what is due auto-reformat and what not.